static/js/chart/draw_bar.ts

```diff
@@ -1058,7 +1062,6 @@ export function drawHorizontalBarChart(
       curr[dataPoint.label] = dataPoint.value;
       curr.dcid = dataPoint.dcid;
       curr.date = dataPoint.date;
-      curr.value = dataPoint.value;
```

Comment:
This used to be used to render values on the tooltips, but is no longer used. `d3.stack()` mangles the value downstream, resulting in incorrect tooltips. Bar lengths and values are already calculated using the value in line 1058.